src/utils.ts

```diff
@@ -156,17 +156,20 @@ export const getCioClient = (apiKey?: string, cioJsClientOptions?: ConstructorCl
 export const getActiveSectionsWithData = (
   activeSections: UserDefinedSection[],
```

Comment:
This is where we have the sections configs and I believe it's the best place we can add the displayName for the recommendations active sections
<img width="352" alt="Screenshot 2024-01-24 at 11 27 15 PM" src="https://github.com/Constructor-io/constructorio-ui-autocomplete/assets/58053149/851c4530-927e-4869-99ad-1bd14a1d38dc">